aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -2709,6 +2709,7 @@ Host Filtering Middleware is disabled by default. To enable the middleware, defi
 
 ## Additional resources
 
+* When targeting linux, app shut down and restart may fail to restart UNIX socket. For more information, see [this GitHub issue](https://github.com/dotnet/aspnetcore/issues/14134).
```

Comment:
I'd suggest something like this, and perhaps also include instructions on how to work around the issue:

```suggestion
* When using UNIX sockets on Linux, the socket is not automatically deleted on app shut down. For more information, see [this GitHub issue](https://github.com/dotnet/aspnetcore/issues/14134).
```